ova/Libraries/provision-opendistro.sh

```diff
@@ -160,8 +150,8 @@ installElasticsearch() {
         if [ ${ram} -eq "0" ]; then
             ram=1;
         fi
-        eval "sed -i "s/-Xms1g/-Xms${ram}g/" /etc/elasticsearch/jvm.options $debug"
-        eval "sed -i "s/-Xmx1g/-Xmx${ram}g/" /etc/elasticsearch/jvm.options $debug"
+        sed -i "s/-Xms1g/-Xms${ram}g/" /etc/elasticsearch/jvm.options $debug
+        sed -i "s/-Xmx1g/-Xmx${ram}g/" /etc/elasticsearch/jvm.options $debug
         jv=$(java -version 2>&1 | grep -o -m1 '1.8.0' )
         if [ "$jv" == "1.8.0" ]
```

Comment:
```suggestion
        if [ "$jv" = "1.8.0" ]
```

Reply:
Done in f1c55144851482b4a51acca472ba0a56429b6b9d